apps/node_monitoring/src/configuration.rs

```diff
@@ -437,6 +439,9 @@ impl DeployMonitoringEnvironment {
                 .unwrap_or("5")
                 .parse::<u64>()
                 .expect("Expected u64 value of seconds"),
+            explorer_url: args
```

Comment:
You haven't introduced this argument to `clap`.

Reply:
thanks, fixed.